### PR TITLE
Set override trace hostname to return instanceId

### PR DESCRIPTION
**Pull request by kraman**
Connect to strongloop-internal/scrum-nodeops#718

### CODE REVIEW

File: lib/service-manager.js [hunk 1]
```diff
@@ -322,6 +322,10 @@ prototype.getInstanceEnv = function(instanceId, callback) {
         );
 
       var env = getServiceEnv(service);
+
+      // Set override env so strong-trace returns the instanceId as the hostname
+      env.STRONGLOOP_HOSTNAME_OVERRIDE = instanceId;
```
Comment (rmg) [inline lib/service-manager.js]: Should this include the service id as well? Like `fmt('%s-%s', serviceId, instanceId);` or something..

Comment (kraman) [inline lib/service-manager.js]: @rmg not necessary. instanceId is unique and easily relatable to service or executor.

Comment (rmg) [inline lib/service-manager.js]: Ok. If it ends up not being unique enough it should be easy and safe to change it later anyway :-)

**Comment by rmg** [comment]
Could possibly use a line in the tests somewhere, but I don't care enough.

LGTM, :shipit:

**Comment by kraman** [comment]
@slnode test please

**Comment by rmg** [comment]
@slnode test please

**Comment by kraman** [comment]
@slnode test please

**Comment by rmg** [comment]
@kraman I'm about to merge #242. If you rebase after it is merged the build should pass.

**Comment by rmg** [comment]
Unless the problem is that this change is causing supervisor to crash, that is.. that would be unfortunate.

**Comment by rmg** [comment]
@slnode test please

**Comment by rmg** [comment]
@slnode test please

**Comment by rmg** [comment]
@slnode test please

**Comment by rmg** [comment]
I really don't like how the environment variable has to be set in 2 places, but the tests are passing now at least...

**Comment by kraman** [comment]
@slnode test please

**Comment by rmg** [comment]
LGTM.

**Pull request merged**